### PR TITLE
added issue and pull request templates

### PR DESCRIPTION
**Pull request by Coinhexa**
Added issue template and pull request template for GitHub

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (fpgmaas) [inline .vscode/settings.json:1]: I'm assuming this file was added by accident? If so, please remove.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+**Expected behavior**
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+**System [please complete the following information]:**
+
+-   OS: e.g. [Ubuntu 18.04]
+-   Language Version: [e.g. Python 3.8]
+-   Virtual environment: [e.g. Conda]
```
Comment (fpgmaas) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: Maybe nit-picky, but using Conda as an example here seems a bit strange. Maybe change to `Poetry version: [e.g. 1.1.13]`

Comment (Coinhexa) [reply .github/ISSUE_TEMPLATE/bug_report.md:29]: no worries, i ll fix that as well :)

**Comment by fpgmaas** [comment]
Thanks for your contribution! You could also consider adding the files to `{{cookiecutter.project_name}}/.github`, so any repository created with the template will also include your added issue templates.

**Comment by fpgmaas** [comment]
It seems a few `.DS_Store` files were also added by accident.

**Comment by Coinhexa** [comment]
I thought the .gitignore blocked .DS_Store files, ah my bad! I ll fix it

**Comment by fpgmaas** [comment]
> I thought the .gitignore blocked .DS_Store files, my bad! I ll fix it

No worries. The changes to the file are still being tracked, so you'll need to manually remove them. See e.g. [here](https://stackoverflow.com/questions/1274057/how-can-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitign)

**Comment by fpgmaas** [comment]
> I thought the .gitignore blocked .DS_Store files, my bad! I ll fix it

The files are still not removed. See the 'Files changed' tab and the link I shared earlier for ways to resolve this.

**Pull request merged**